tests/kernel/fp_sharing/generic/src/main.c

```diff
@@ -233,7 +239,7 @@ void load_store_low(void)
 		 * After every 1000 iterations (arbitrarily chosen), explicitly
 		 * disable floating point operations for the task.
 		 */
-#if (defined(CONFIG_X86) && defined(CONFIG_LAZY_FP_SHARING)) || \
+#if (defined(CONFIG_X86) && defined(CONFIG_LAZY_FP_SHARING)) ||	\
```

Comment:
why this change?

Reply:
I believe this was done by the uncrustify tool. There are a few other changes that crept in due to me running it on the files I changed.